bigbone/src/main/kotlin/social/bigbone/MastodonClient.kt

```diff
@@ -271,6 +275,8 @@ private constructor(
         private val gson = Gson()
         private var accessToken: String? = null
         private var debug = false
+        private val baseUrlV1 = "https://$instanceName/api/v1"
```

Comment:
Do these need to be defined outside of getInstanceVersion()?

Reply:
Fixed 🙂